### PR TITLE
Fixed retail task #9, #16, #20, #35

### PR DESCRIPTION
**Pull request by dayyyyyyyyyy**
Hello, we are opening this PR to inform you that we have found few more potential errors in retail tasks. This PR is an extension of previous PR [#2 ](https://github.com/sierra-research/tau-bench/pull/2)

The revisions include:

- Tasks with incorrect action arguments
- Tasks with ambiguous or incorrect instructions

We appreciate your effort for reviewing these revisions and maintaining this project.

@ysymyth @SamWarmuth @karthikncode @pedramr

### CODE REVIEW

File: tau_bench/envs/retail/tasks.py [hunk 1]
```diff
@@ -262,7 +262,7 @@
                 "arguments": {
                     "order_id": "#W6390527",
                     "item_ids": ["8384507844"],
-                    "new_item_ids": ["4385534692"],
+                    "new_item_ids": ["7624783998"],
```
Comment (dayyyyyyyyyy) [inline tau_bench/envs/retail/tasks.py:265]: Task #9
User wants a brighter lamp and prefers AC adapter over battery and USB.
There are two items (4385534692 and 7624783998, colored white and black, respectively) that fit these conditions.
Original GT action suggests item ID 4385534692, which is unavailable.

File: tau_bench/envs/retail/tasks.py [hunk 2]
```diff
@@ -419,7 +419,7 @@
     {
         "annotator": 0,
         "user_id": "fatima_johnson_7581",
-        "instruction": "You are Fatima Johnson in 78712. You want to cancel all pending orders and return the watch you have received (but nothing else), and you want to know the total amount you can get back. You are a private person that does not want to reveal much about yourself.",
+        "instruction": "You are Fatima Johnson in 78712. You want to cancel all pending orders (since they are no longer needed) and return the watch you have received (but nothing else), and you want to know the total amount you can get back. You are a private person that does not want to reveal much about yourself.",
```
Comment (dayyyyyyyyyy) [inline tau_bench/envs/retail/tasks.py:422]: Task #16
Reason for cancelling was omitted.

File: tau_bench/envs/retail/tasks.py [hunk 3]
```diff
@@ -527,7 +527,7 @@
     {
         "annotator": 0,
         "user_id": "ethan_garcia_1261",
-        "instruction": "You are Ethan Garcia, and you live in Denver, 80280. You just won a lottery, and you want to upgrade all your items to the most expensive options (but make sure the shoe is still the same size). You are happy to pay the difference with your GC, but otherwise PayPal is fine. You are a mysterious person and do not want to reveal much about yourself.",
+        "instruction": "You are Ethan Garcia, and you live in Denver, 80280. You just won a lottery, and you want to upgrade all your items to the most expensive options (but make sure the shoe is still the same size). You want to pay the difference with your GC, but if it is impossible, PayPal is fine. You are a mysterious person and do not want to reveal much about yourself.",
```
Comment (dayyyyyyyyyy) [inline tau_bench/envs/retail/tasks.py:530]: Task #20
Original instruction is ambiguous about payment selection.

File: tau_bench/envs/retail/tasks.py [hunk 4]
```diff
@@ -1005,7 +1005,7 @@
     {
         "annotator": 0,
         "user_id": "aarav_santos_2259",
-        "instruction": "You are aarav_santos_2259 and aarav.santos8321@example.com and aarav.santos8320@example.com. You want to return the speaker that is more expensive yet not resistent to water. Also, You want to modify the 17-inch laptop to the 13-inch version in the same order. If no exact item is available, you want to know all available 13-inch options, and you prefer i5 over i7, and prefer silver and black than other colors. You are a rude person.",
+        "instruction": "You are aarav_santos_2259 and aarav.santos8321@example.com and aarav.santos8320@example.com. You want to return the speaker that is more expensive yet not resistent to water. Also, You want to modify the 17-inch laptop to the 13-inch version in another order. If no exact item is available, you want to know all available 13-inch options, and you prefer i5 over i7, and prefer silver and black than other colors. You are a rude person.",
```
Comment (dayyyyyyyyyy) [inline tau_bench/envs/retail/tasks.py:1008]: Task #35
Speaker and 17-inch laptop are not in the same order.

**Comment by karthikncode** [comment]
Thanks for the fixes!

**Pull request merged**